src/build/build-libgerbil.ss

```diff
@@ -386,24 +388,31 @@
     (displayln "... build " libgerbil)
     (let (libgerbil-ldd (filter (? (not string-empty?)) (string-split ld-options #\space)))
       (if (eq? mode 'shared)
-        (invoke-gcc ["-shared" "-o" libgerbil
-                     libgerbil-ldd ...
-                     static-module-o-paths ...
-                     builtin-modules-o-path
-                     link-o-path])
+        (cond-expand
+          (darwin (invoke-gcc ["-dynamiclib" "-o" libgerbil "-install_name"
+                               (string-append (getenv "GERBIL_PREFIX") "/lib/libgerbil.dylib")
+                               libgerbil-ldd ...
+                               static-module-o-paths ...
+                               builtin-modules-o-path
+                               link-o-path]))
+          (else (invoke-gcc ["-shared" "-o" libgerbil
+                             libgerbil-ldd ...
+                             static-module-o-paths ...
+                             builtin-modules-o-path
+                             link-o-path])))
         (invoke-ar ["cq" libgerbil
                     static-module-o-paths ...
                     builtin-modules-o-path
                     link-o-path]))
       (call-with-output-file (string-append libgerbil ".ldd")
-	(cut write
-	     (filter
-	      (cond-expand
-		(darwin
-		 (lambda (arg) (not (string-prefix? (string-append "-L" (gerbil-libdir)) arg))))
-		(else true))
-	      libgerbil-ldd)
-	     <>)))
+        (cut write
+             (filter
+              (cond-expand
+                (darwin
+                 (lambda (arg) (not (string-prefix? (string-append "-L" (gerbil-libdir)) arg))))
+                (else true))
+              libgerbil-ldd)
+             <>)))
```

Comment:
is this a tab -vs- spaces thing? We use spaces in general, as tabs are unpredictable.

Reply:
Yes, the original file contains tab which I did not think were right, so I took the liberty to untabify the file.

Reply:
great, thank you!